### PR TITLE
Step 9. Configure CI/CD, code quality

### PR DESCRIPTION
**Pull request by yahormarashchuk**
comapre two test and create file build.yml

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+# run arbitrary code.
+extension-pkg-whitelist=
+
+# Specify a score threshold to be exceeded before program exits with error.
+fail-under=10.0
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=CVS,
+      mange.py,
```
Comment (inauros) [inline .pylintrc]: Did you mean manage.py? :)

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,515 @@
...
+
+# Number of spaces of indent required inside a hanging or continued line.
+indent-after-paren=4
+
+# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
+# tab).
+indent-string='    '
+
+# Maximum number of characters on a single line.
+max-line-length=100
```
Comment (inauros) [inline .pylintrc]: Please replace 100 with 120

**Review by inauros — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,5 @@
+[![pylint_test](https://github.com/EgorHubBrest/Human_Resourse_Managment/actions/workflows/pylint.yml/badge.svg)](https://github.com/EgorHubBrest/Human_Resourse_Managment/actions/workflows/pylint.yml)
+[![coveralls_test](https://github.com/EgorHubBrest/Human_Resourse_Managment/actions/workflows/coveralls_test.yml/badge.svg?branch=developer)](https://github.com/EgorHubBrest/Human_Resourse_Managment/actions/workflows/coveralls_test.yml)
```
Comment (inauros) [inline README.md]: Please replace the "pylint_test" and "coveralls_test" workflows with the "build" workflow

**Review by inauros — CHANGES_REQUESTED** [review]

File: .pylintrc (hunk 1 above)
Comment (yahormarashchuk) [reply .pylintrc]: Its my mistake),I will corecte it.

File: .pylintrc (hunk 2 above)
Comment (yahormarashchuk) [reply .pylintrc]: Ok,Im doing it

File: README.md (hunk 3 above)
Comment (yahormarashchuk) [reply README.md]: Ok,I will change

**Pull request merged**